src/admin-views/help-hub.php

```diff
@@ -36,17 +36,24 @@ class="tribe-events-admin-title__logo"
 		/>
 		Help
 	</h1>
-	<dialog id="tec-settings-nav-modal" class="tec-settings-form__modal">
+	<dialog id="tec-settings-nav-modal" class="tec-settings-form__modal" aria-labelledby="tec-settings-nav-modal-title" aria-modal="true" role="dialog">
```

Comment:
`aria-labelledby` or `aria-describedby`? Iffy here, there's not a lot of content. I'll leave that up to you.